presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -643,6 +651,9 @@ void PrestoServer::registerCustomOperators() {
       std::make_unique<facebook::presto::operators::ShuffleWriteTranslator>());
   facebook::velox::exec::Operator::registerOperator(
       std::make_unique<operators::ShuffleReadTranslator>());
+  facebook::velox::exec::Operator::registerOperator(
+      std::make_unique<
+          facebook::presto::operators::BroadcastWriteTranslator>());
```

Comment:
This is POS-specific, right? Let's split Prestissimo and POS into 2 separate classes to avoid mixing the logic.

Reply:
Good point. I was following existing shuffle registration. Should I refactor shuffle ones too ? Is it fine to do this refactoring in seperate PR?

Reply:
Yes, let's refactor in a separate PR (a prerequisite to this one). Thanks.

Reply:
Added TODO for splitting presto and pos server